data/json/items/vehicle/engine.json

```diff
@@ -86,6 +86,18 @@
     "displacement": 20,
     "faults": [  ]
   },
+  {
+    "id": "aero_engine_light",
+    "copy-from": "engine_gasoline",
+    "type": "ENGINE",
+    "name": { "str": "light aero-engine" },
+    "description": "An air-cooled, four-cylinder, horizontally opposed internal combustion engine, rated for 150 horsepower.  Commonly used on light aircraft.",
```

Comment:
This looks fine as far as it goes, but is it based on any particular aircraft engine?  The Lycoming O-235 used in Cessna 152s is a basically like this, but is 75-100 kW (100-135 HP) depending on the version.

Reply:
It's based on Lycoming O-320. I have a Wikipedia link for it in the post above, which says they're either 150 or 160 HP.

Reply:
Sorry, I totally missed that.  Looks good then.